docs/troubleshooting.asciidoc

```diff
@@ -0,0 +1,236 @@
+[id="{p}-troubleshooting"]
+== Troubleshoot a failing cluster
+
+When things don't seem to work as expected, there are several places you can look at to investigate.
+
+[float]
+[id="{p}-get-resources"]
+=== Get the list of resources
+
+To deploy and manage the Elastic stack, ECK creates several resources in the namespace the main resource is deployed in.
+
+For example, each Elasticsearch node and Kibana instance has a dedicated Pod.
+Check the Pods running and their status, to compare with the expected instances:
+
+[source,sh]
+----
+kubectl get pods
+
+NAME                                 READY     STATUS    RESTARTS   AGE
+elasticsearch-sample-es-66sv6dvt7g   0/1       Pending   0          3s
+elasticsearch-sample-es-9xzzhmgd4h   1/1       Running   0          27m
+elasticsearch-sample-es-lgphkv9p67   0/1       Pending   0          3s
+kibana-sample-kb-5468b8685d-c7mdp    0/1       Running   0          4s
+----
+
+Check the services:
+
+[source,sh]
+----
+kubectl get services
+
+NAME                           TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+elasticsearch-sample-es-http   ClusterIP   10.19.248.93    <none>        9200/TCP   2d
+kibana-sample-kb-http          ClusterIP   10.19.246.116   <none>        5601/TCP   3d
+----
+
+[float]
+[id="{p}-describe-failing-resources"]
+=== Describe failing resources
+
+If an Elasticsearch node does not come up, it is possible the associated Pod cannot be scheduled by Kubernetes.
+You may see this by looking at the Status of the Pods. If it does not become `Running` after a few seconds, chances are something's wrong:
+
```

Comment:
> If an Elasticsearch node does not come up, it's probably because Kubernetes cannot schedule the associated Pod. Check the Pod status: if it's not `Running` after a few seconds, something is wrong: